lib/models/states/user_state.dart

```diff
@@ -53,37 +60,12 @@ class UserState {
 
   dynamic toJson() => {
     'id': id,
+    'type': parseUserTypeToString(type),
     'isFtu': isFtu,
     'authStatus': parseAuthStatusToString(authStatus),
     'saveProfileState': saveProfileState.toJson()
   };
 
-  static String parseAuthStatusToString(AuthStatus authStatus) {
-    switch (authStatus) {
-      case AuthStatus.notDetermined:
-        return "notDetermined";
-      case AuthStatus.notLoggedIn:
-        return "notLoggedIn";
-      case AuthStatus.loggedIn:
-        return "loggedIn";
-      default:
-        return "notDetermined";
-    }
-  }
-
-  static AuthStatus parseAuthStatusToValue(String authStatus) {
-    switch (authStatus) {
-      case "notDetermined":
-        return AuthStatus.notDetermined;
-      case "notLoggedIn":
-        return AuthStatus.notLoggedIn;
-      case "loggedIn":
-        return AuthStatus.loggedIn;
-      default:
-        return AuthStatus.notDetermined;
-    }
-  }
-
   @override
```

Comment:
Why did u delete those lines?

Reply:
I moved them into the `enum_parser.dart` file!

Reply:
Yep! I understand now!